api/utils/json_response.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+""" This module contains the Json response class
+"""
+from enum import Enum
+from json import dumps
+from fastapi import status
+from fastapi.responses import JSONResponse
+from fastapi.encoders import jsonable_encoder
+
+
+class RequestStatus(Enum):
+    SUCCESS = "success"
+    ERROR = "error"
```

Comment:
Not necessary for the accepted convention

Reply:
Should I remove it?

Reply:
Yes